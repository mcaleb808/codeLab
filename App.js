import { createStackNavigator, createAppContainer } from "react-navigation";
import HomeScreen from "./src/screens/HomeScreen";

const MainNavigator = createStackNavigator(
  {
    Home: { screen: HomeScreen }
  },
  {
    headerMode: "none"
  }
);

const App = createAppContainer(MainNavigator);

export default App;
