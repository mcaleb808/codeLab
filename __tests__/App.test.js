import "react-native";
import React from "react";
import App from "../App";
import renderer from "react-test-renderer";

describe("App snapshot", () => {
  it("renders the app", async () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
