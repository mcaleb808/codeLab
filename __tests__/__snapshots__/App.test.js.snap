// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders the app 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    enabled={false}
    hitSlop={
      Object {
        "right": 50,
      }
    }
    maxDeltaY={20}
    minOffsetX={5}
    onGestureEvent={[Function]}
    onHandlerStateChange={[Function]}
  >
    <View
      style={
        Object {
          "backgroundColor": "#eee",
          "flex": 1,
          "flexDirection": "column-reverse",
          "overflow": "hidden",
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          pointerEvents="auto"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "transform": Array [
                Object {
                  "translateX": 0,
                },
                Object {
                  "translateY": 0,
                },
              ],
            }
          }
        >
          <View
            accessibilityElementsHidden={false}
            style={
              Object {
                "backgroundColor": "#fff",
                "flex": 1,
              }
            }
          >
            <RCTSafeAreaView>
              <View
                accessibilityIgnoresInvertColors={true}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#F8F8F8",
                    "borderBottomColor": "#f2f2f2",
                    "borderBottomWidth": 0.5,
                    "flexDirection": "row",
                    "height": 64,
                    "justifyContent": "space-between",
                    "paddingHorizontal": 10,
                    "paddingTop": 20,
                  }
                }
              >
                <Image
                  style={
                    Array [
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      },
                      Object {
                        "height": 64,
                        "width": undefined,
                      },
                      undefined,
                    ]
                  }
                  testID="headerContainer"
                />
                <View
                  style={
                    Object {
                      "alignItems": "flex-start",
                      "flex": 1,
                    }
                  }
                />
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flex": 3,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "color": "#1F2A34",
                        "fontSize": 20,
                        "fontWeight": "bold",
                      }
                    }
                    text="Developers"
                    theme={
                      Object {
                        "colors": Object {
                          "disabled": "hsl(208, 8%, 90%)",
                          "divider": "#bcbbc1",
                          "error": "#ff190c",
                          "grey0": "#393e42",
                          "grey1": "#43484d",
                          "grey2": "#5e6977",
                          "grey3": "#86939e",
                          "grey4": "#bdc6cf",
                          "grey5": "#e1e8ee",
                          "greyOutline": "#bbb",
                          "platform": Object {
                            "android": Object {
                              "error": "#f44336",
                              "primary": "#2196f3",
                              "secondary": "#9C27B0",
                              "success": "#4caf50",
                              "warning": "#ffeb3b",
                            },
                            "ios": Object {
                              "error": "#ff3b30",
                              "primary": "#007aff",
                              "secondary": "#5856d6",
                              "success": "#4cd964",
                              "warning": "#ffcc00",
                            },
                          },
                          "primary": "#2089dc",
                          "searchBg": "#303337",
                          "secondary": "#8F0CE8",
                          "success": "#52c41a",
                          "warning": "#faad14",
                        },
                      }
                    }
                  >
                    Developers
                  </Text>
                </View>
                <View
                  style={
                    Object {
                      "alignItems": "flex-end",
                      "flex": 1,
                    }
                  }
                />
              </View>
              <View>
                <ActivityIndicator
                  animating={true}
                  color="#1F2A34"
                  hidesWhenStopped={true}
                  size="large"
                />
              </View>
            </RCTSafeAreaView>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
